### DISCUSSION TITLE
Has there been a functional update in the near future?

### DISCUSSION THREAD

**Discussion by kentzh1** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#730] [General]
No feature updates for more than 2 months. Have you considered updating these functions? Looking forward to the application getting better and better.

**Comment by MatthiasKerbl** [comment]
> No feature updates for more than 2 months. Have you considered updating these functions? Looking forward to the application getting better and better.

Hi @kentzh1,

We had a lot going on over the last couple of months. However, we are working on a couple of new features in the background and hopefully one of them will be ready soon to be released.

Cheers,
Matthias

**Reply by loughrey101** [reply]
I find that when you set it to a certain percentage it won't drop the battery to that percentage automatically and often won't stay at the set target in the laest MacOS using the 16 inch M1 Pro.
